### PR TITLE
BaSP-TG-75: Project form

### PR DESCRIPTION
**Pull request by CarVigil**

### CODE REVIEW

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
The tittle and the headers of the table should be this color "#373867". The form doesnt accept project without an end date.

**Review by Facund0Fernandez — COMMENTED** [review]
The selector is not displayed correctly.
select

**Review by GuidoCerioni — APPROVED** [review]
Everything seem fine. The Select component doesnt shows properly, but its a problem with the width of the shared component. i think we will fix it later

File: src/Components/Project/Project.module.css [hunk 1]
```diff
@@ -4,27 +4,69 @@
...
+.employee {
+  display: flex;
+  flex-direction: column;
+  padding: 25px;
+  border: solid 1px #373867;
+}
+.select{
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 15px;
```
Comment (Facund0Fernandez) [inline src/Components/Project/Project.module.css:41]: Trying with a width here maybe the display of the Select could be solved.

**Review by Facund0Fernandez — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Review by Gearyandres94 — APPROVED** [review]
Great Job!!

**Pull request merged**